src/Common/Config/ConfigProcessor.cpp

```diff
@@ -171,6 +200,70 @@ static void mergeAttributes(Element & config_element, Element & with_element)
     with_element_attributes->release();
 }
 
+#if USE_SSL
+
+std::string ConfigProcessor::encryptValue(const std::string & codec_name, const std::string & value)
+{
+    auto codec = DB::CompressionCodecEncrypted(getEncryptionMethod(codec_name));
+
+    DB::Memory<> memory;
```

Comment:
As far as I see, `DB::Memory<>` is optimized for max performance (alignment, no memset). Is that really needed here or can we simplify and use a standard C++ heap buffer?

Reply:
I use `DB::Memory<>` because it's also used with encryption codecs in decompress fuzzer:
https://github.com/ClickHouse/ClickHouse/blob/master/src/Compression/fuzzers/encrypted_decompress_fuzzer.cpp#L290C1-L293C25